### PR TITLE
upgrade: crates

### PR DESCRIPTION
**Pull request by ry**

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -2047,6 +2100,12 @@ dependencies = [
  "winapi-util",
 ]
 
+[[package]]
+name = "scoped-tls"
```
Comment (piscisaureus) [inline Cargo.lock:2104]: 2 copies of scoped-tls now

File: Cargo.lock [hunk 2]
```diff
@@ -537,22 +540,42 @@ dependencies = [
...
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "8e93d7f5705de3e49895a2b5e0b8855a1c27f080192ae9c32a6432d50741a57a"
+dependencies = [
+ "libc",
+ "redox_users",
+ "winapi 0.3.8",
+]
+
+[[package]]
+name = "dirs-sys-next"
```
Comment (piscisaureus) [inline Cargo.lock:564]: dirs, dirs-next, dirs-sys, dirs-sys-next...

File: Cargo.lock [hunk 3]
```diff
@@ -2223,6 +2282,22 @@ dependencies = [
  "url 1.7.2",
 ]
 
+[[package]]
+name = "sourcemap"
```
Comment (piscisaureus) [inline Cargo.lock:2286]: two versions of 'sourcemap' now

Comment (ry) [reply Cargo.lock]: fixed

File: Cargo.lock (hunk 1 above)
Comment (ry) [reply Cargo.lock:2098]: can't dedup that unfortunately

File: Cargo.lock (hunk 2 above)
Comment (ry) [reply Cargo.lock:564]: yeah - it's ridiculous. All for something to tell us where the cache dir is. We should drop this dep.

Comment (ry) [reply Cargo.lock]: I was able to get rid of most of this... will remove the dirs dep entirely later in https://github.com/denoland/deno/pull/6385

**Pull request merged**